src/scss/sections/_hero-banner.scss

```diff
@@ -0,0 +1,72 @@
+.hero-carousel {
+
+  .offcanvas-active & {
+    .carousel {
+      z-index: -1;
```

Comment:
absolutely no this.
1. we have a class for different z-index
2. we should rather manage the z-index of the header

Reply:
move this to header.scss
but i have tried with set index carousel to use $zindex-dropdown (lowest var value from bootstrap), but it doesn't works even when i try with value 0 or 1, so only negative its working.